lib/client.js

```diff
@@ -25,134 +35,135 @@ exports.create = function (config) {
           'Content-Length': (data) ? Buffer.byteLength(data, 'utf8') : 0,
           'User-Agent': 'recurly-js/' + pjson.version
         }
-      };
+      }
 
       if (headers) {
-        for (var p in headers) {
-          if (!headers.hasOwnProperty(p)) continue;
+        for (const p in headers) {
```

Comment:
Shouldn't this be `let`? Variable `p` would be getting reassigned in all iterations.